test/executables/test_components/file_with_comment/react_render_with_comment.dart

```diff
@@ -0,0 +1,12 @@
+import "package:react/react.dart" as react;
+import "package:react/react_dom.dart" as react_dom;
+import "package:react/react_client.dart";
+import "dart:html";
+
```

Comment:
add a new file `react_faker.dart` or something in the `test_components` dir with this content:
```
render(dynamic arg1, dynamic arg2) {};
```
then do `import '../react_faker.dart' as react_dom;` in each of these files and just put like a string in the first 2 args

greg did mention these files don't even need to static analyze but then we would have to ignore them so id prefer this if everyone doesn't mind lol